content/docs/get-started/index.md

```diff
@@ -279,238 +246,258 @@ Alternatively, you could save the dataset itself
 </tab>
 </toggle>
 
-We saw that MLEM provides a consistent and friendly way for you to work with
-models both via API and CLI. However, MLEM **really** shines when you need to
-package and deploy your models, either as part of an application or locally for
-testing.
+Ok, we saw how easy working with models is, both via API and CLI. However, MLEM
+**really shines** when you need to package and deploy your models! (⌐■_■)
 
-## Deploying and Serving models
+## Serving models locally
 
-MLEM can serve a model for you using different server implementations, for
-example FastAPI or RabbitMQ. Here we'll check out how it works with FastAPI
-since serving models via a REST API is a very common use case.
+MLEM can serve any model for you using different server implementations like
+FastAPI, RabbitMQ or Streamlit. Let's check out how it works with FastAPI since
```

Comment:
```suggestion
[FastAPI](/doc/user-guide/serving/fastapi), [RabbitMQ](/doc/user-guide/serving/rabbitmq) or [Streamlit](/doc/user-guide/serving/streamlit). Let's check out how it works with FastAPI since
```